<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Part 2 machines</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEditor">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Editor</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditEditor">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>C:\WindowsNotepad.exe</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditMessages">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>611</width>
      <height>181</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Messages</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAltiumPart2Machines">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Altium part 2 machines
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPanasonicCoordsToOriginal">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Panasonic coordinates to original</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWorkDirectory">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Work directory&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditWorkDirectory">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>664</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDE_part_to_machines">
    <property name="title">
     <string>DE part to machines</string>
    </property>
   </widget>
   <addaction name="menuDE_part_to_machines"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
